builds/misc/packages.yaml

```diff
@@ -99,3 +99,54 @@ jobs:
           PathtoPublish: '$(build.artifactstagingdirectory)'
           ArtifactName: 'iotedged-$(os)-$(arch)'
         condition: succeededOrFailed()
+
+################################################################################
+  - job: mariner_linux
+################################################################################
+    displayName: Mariner_Linux
+    pool:
+      vmImage: 'ubuntu-18.04'
+    variables:
+      NugetSecurityAnalysisWarningLevel: warn
+    strategy:
+      matrix:
+        CBL-Mariner1.0-amd64:
+          arch: amd64
+          os: mariner
+          release: tags/1.0-stable
+          target.iotedged: builds/mariner/out/RPMS/x86_64/
+    steps:
+      - bash: |
+          BASE_VERSION=`cat $BUILD_SOURCESDIRECTORY/edgelet/version.txt`
+          VERSION="$BASE_VERSION$BUILD_BUILDNUMBER"
+          echo "##vso[task.setvariable variable=VERSION;]$VERSION"
+          echo "PACKAGE_ARCH=$(arch)"
+          echo "PACKAGE_OS=$(os)"
+          echo "##vso[task.setvariable variable=MARINER_RELEASE;]$(release)"
+        displayName: Set Version
+      - script: chmod +777 edgelet/build/linux/package-mariner.sh
+        displayName: make package executable
```

Comment:
I believe git actually preserve the original permission of the file. So if you check the file in as u+rx, it should just work here? 
Or.... you can even explicitly tell git to do so 
```
git update-index --chmod=+x package-mariner.sh
```
Then you can `git commit` and push per usual.